jayanatl/vim_python
# Nvim (#2)

* create skeleton

* add basic config

* fix python3 install

* add install_centos.sh

* update readme

* bugfix: install centos file as normal user

* bugfix permissions

* few bug fixes

* check before updating bashrc

* fix temp folder

* clear stdin before reading data

* test read

* read test

* read test

* update readme

* update readme

* update readme

* update readme

* Fix readme

* Create LICENSE

* update readme

* update readme

* Add sanity check

* fixes related repo rename

* Install virtualenvwrapper

* nvim basic config

* nvim basic setup

* bugfix

* bugfix

* bugfix

* add powerline

* ignore error while vim:pluginstall

* remove clear

* bugfix

* update readme

* formating fix

* fix formatting

* fix formatting

* Add vim helper

* Install ctags

* vim script update

* add window switch keys

* remove use of venv and use pip --user instead

* update host_python

* Disabled git templates

* update setup

* bugfix

* add isort and yapf

* bugfix

* update leader e to force quit

* add flake config

* update gitconf

* Create add_trusted_ca

* Rename add_trusted_ca to add_trusted_ca.md

* Update and rename add_trusted_ca.md to Post_Install.md

* Update Post_Install.md

* Update Post_Install.md

* Update Post_Install.md

* Update Post_Install.md

* Update TODO list

* Update .profile

* Update .profile

* Update setup_centos.sh

* update nvim init

* add tmux config
